### PR TITLE
docs: small updates to astra vectorize docs

### PR DESCRIPTION
**Pull request by jordanrfrazier**

### CODE REVIEW

File: src/backend/base/langflow/components/embeddings/AstraVectorize.py [hunk 1]
```diff
@@ -50,37 +50,38 @@ class AstraVectorize(Component):
...
             info=f"The embedding model to use for the selected provider. Each provider has a different set of models "
-            f"available (full list at https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
+            f"available (https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
             required=True,
         ),
+        SecretStrInput(
+            name="provider_api_key",
+            display_name="Provider API Key",
+            info="An alternative to the Astra Authentication that passes an API key for the provider with each request to Astra DB. This may be used when Vectorize is configured for the collection, but no corresponding provider secret is stored within Astra's key management system.",
+        ),
```
Comment (jordanrfrazier) [inline src/backend/base/langflow/components/embeddings/AstraVectorize.py:69]: I made this appear by default. The reasoning is that 
a) This is the quickest path to testing out vectorize on a new collection
b) If the collection is already configured for vectorize, but the key was not stored in astra, this is (by my guess) the most common pattern for authentication.

File: src/backend/base/langflow/components/embeddings/AstraVectorize.py [hunk 2]
```diff
@@ -50,37 +50,38 @@ class AstraVectorize(Component):
...
         MessageTextInput(
             name="api_key_name",
-            display_name="API Key name",
+            display_name="Provider API Key Name",
             info="The name of the embeddings provider API key stored on Astra. If set, it will override the 'ProviderKey' in the authentication parameters.",
+            advanced=True,
```
Comment (jordanrfrazier) [inline src/backend/base/langflow/components/embeddings/AstraVectorize.py:74]: Is there a common case where the `providerKey` is set differently than the default?

Comment (nicoloboschi) [reply src/backend/base/langflow/components/embeddings/AstraVectorize.py:74]: what do you mean? there's no default value. The value is defined by the user

File: src/backend/base/langflow/components/vectorstores/AstraDB.py [hunk 3]
```diff
@@ -116,6 +116,7 @@ class AstraVectorStoreComponent(LCVectorStoreComponent):
             name="embedding",
             display_name="Embedding or Astra Vectorize",
             input_types=["Embeddings", "dict"],
+            info="Allows either an embedding model or an Astra Vectorize configuration. If Astra Vectorize is already configured for the collection, this field is not required.",
```
Comment (nicoloboschi) [inline src/backend/base/langflow/components/vectorstores/AstraDB.py:119]: did you try what happens if you don't set it ? 
I believe the AstraDBVectorStore requires on of those.

File: src/backend/base/langflow/components/embeddings/AstraVectorize.py (hunk 2 above)
Comment (jordanrfrazier) [reply src/backend/base/langflow/components/embeddings/AstraVectorize.py:74]: This is the key name for the authentication token, correct? e.g.
```
CollectionVectorServiceOptions(
           ...
            authentication={
                "providerKey": os.environ["OPENAI_API_KEY"],
           }
        )
```

But going through the UI flow, I see that you are correct in your other comment that the user-defined key name is required during creation. So I will undo these changes, and have `Provider API Key Name` shown by default, and `Provider API Key` in Advanced.

**Review by zzzming — APPROVED** [review]

File: src/backend/base/langflow/components/vectorstores/AstraDB.py (hunk 3 above)
Comment (jordanrfrazier) [reply src/backend/base/langflow/components/vectorstores/AstraDB.py:119]: Ugh, yeah. I guess that's something to figure out. It's required by langchain-astradb to have one or the other at the moment.

Comment (jordanrfrazier) [reply src/backend/base/langflow/components/vectorstores/AstraDB.py:119]: Perhaps we could loosen that constraint iff the setup mode is `SetupMode.OFF`?

**Pull request merged**